### DISCUSSION TITLE
Public Agent Search page

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#5373] [Ideas]
Currently the public agent search page only has one search option. Does it need more? and if so what?

**Comment by ewommack** [comment]
Maybe:
used by collection
Agent Type
?

**Comment by Jegelewicz** [comment]
How about all of the things you can see in a public agent page?

**Comment by ewommack** [comment]
I was thinking about some of them, and wondering if allowing search on born and died maybe should not, as relates to https://github.com/ArctosDB/internal/issues/199?

**Comment by dustymc** [comment]
Moving to discussions pending policy resolution and a compelling use case.
